examples/extensions/localserver/local_server.py

```
@@ -83,7 +83,8 @@ def tokenize(self, input_text: str, parameters: TextTokenizationParameters) -> T
 print(heading("Use a local server with a custom model"))
 
 # Instantiate the Local Server with your model
-server = LocalLLMServer(models=[FlanT5Model])
+# set interface="0.0.0.0" to listen on all interfaces
+server = LocalLLMServer(models=[FlanT5Model], interface="127.0.0.1")
```

I've seen some examples where binding to `0.0.0.0` silently failed.